python/GafferImageUI/ImageViewUI.py

```diff
@@ -326,18 +325,18 @@ def getToolTip( self ) :
 
 		return result
 
-	def _updateFromPlug( self ) :
-
-		with self.getContext() :
-			value = self.getPlug().getValue()
+	def _updateFromValues( self, values, exception ) :
 
+		value = sole( values )
```

Comment:
I'm going to leave as-is for now, since `sole()` isn't something new to this PR. But I do agree that it was somewhat out of character for me to do it this way. I think it's because I dearly wanted it to be a Python built-in. Happy to chat afterwards about where it might more sensibly go...